index.html

```diff
@@ -0,0 +1,264 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="index.css" rel="stylesheet" />
+    <link
+      href="//db.onlinewebfonts.com/c/d14035f6b1afeabafbee4abb2ebf0fc6?family=COCOGOOSE"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    
+    <title>Capstone1</title>
+  </head>
+  <body>
+    <header>
+      <nav class="tools">
+        <img src="images/facebook-icon.png" alt="" class="fb_icon">
+        <img src="images/twitter-icon.png" alt="" class="fb_icon">
+        <a href="#" class="english">English</a>
+        <a href="#" class="english">My page</a>
+        <a href="#" class="english" id="last_tool">Logout</a>
+      </nav>
+      <nav id="NavMobile">
+        <img src="images/Hamburger.png" alt="" class="hamburger" id="hamburger1"/>
+        <img src="images/x-icon.png" alt="" class="hamburger" id="hamburger2"/>
+        <ul class="ul_nav" id="ul_nav">
+          <li id="home"><a class="links_Nav" >Home</a></li>
+          <li id="about"><a class="links_Nav" >About</a></li>
+          <li><a class="links_Nav" >Join</a></li>
+          <li><a class="links_Nav" href="#">Program</a></li>
+          <li><a class="links_Nav" href="#">Sponsor</a></li>
+          <li><a class="links_Nav" href="#">News</a></li>
+        </ul>
```

Comment:
- [ ] Kindly make the mobile menu take the full height of the page and stop scrolling when the menu opens for a better UI experience.